app/models/concerns/tristateable.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Reads a column, bypassing activerecord field accessors, so that we can
+# differentiate between nil values and boolean false (by default nils in
+# activerecord are converted to false). Naturally, booleans can only hold
+# true/false, so these fields should be created as strings.
+#
+module Tristateable
```

Comment:
Not sure we need the Tristateable stuff, the DS reads in strings as booleans now and will produce T/F/nil for any column used in the institutions table (data_csv).